content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -21,6 +21,10 @@ a *kubeconfig file*. This is a generic way of referring to configuration files.
 It does not mean that there is a file named `kubeconfig`.
 {{< /note >}}
 
+{{< warning >}}
+Using unverified kubeconfig files could result in malicious code execution and file exposure. 
```

Comment:
You're also impacted if you _generate_ kubeconfig files include data provided from users

Reply:
@hyakuhei, do you mean that if you generate and then use a kubeconfig file based upon untrusted user input, these same bad things could happen to you? Agreed.